src/xrpld/app/misc/AMMHelpers.h

```diff
@@ -51,6 +51,8 @@ reduceOffer(auto const& amount)
 
 }  // namespace detail
 
+enum class IsDeposit : bool { Yes, No };
```

Comment:
Align enum semantics with `bool`

```suggestion
enum class IsDeposit : bool { Yes = true, No = false };
```